packages/material-ui/src/styles/createPalette.js

```diff
@@ -43,6 +43,10 @@ export const light = {
     disabled: 'rgba(0, 0, 0, 0.26)',
     // The background color of a disabled action.
     disabledBackground: 'rgba(0, 0, 0, 0.12)',
+    disabledOpacity: 0.38,
```

Comment:
Were these keys first introduced here in v4.9.3? Is it stable (i.e. can I use it when styling)?

Reply:
We plan to use theses keys more broadly. I think that it's safe to use.